vertical-pod-autoscaler/pkg/utils/vpa/api.go

```diff
@@ -109,8 +109,8 @@ func PodLabelsMatchVPA(podNamespace string, labels labels.Set, vpaNamespace stri
 	return vpaSelector.Matches(labels)
 }
 
-// stronger returns true iff a is before b in the order to control a Pod (that matches both VPAs).
-func stronger(a, b *vpa_types.VerticalPodAutoscaler) bool {
+// Stronger returns true iff a is before b in the order to control a Pod (that matches both VPAs).
```

Comment:
```suggestion
// Stronger returns true if a is before b in the order to control a Pod (that matches both VPAs).
```

A small typo  in the original comment

Reply:
`iff` is actually used in many places in the source code. I am not sure if it is a typo or refers to `if and only if (as a written abbreviation).`

Reply:
That's fair, I think you're right